src/main/java/org/reactome/addlinks/AddLinks.java

```diff
@@ -242,7 +242,8 @@ private String checkLinksForRefDB(LinkCheckManager linkCheckManager, GKInstance
 			}
 
 		}
-		Map<String, LinkCheckInfo> checkedLinks = linkCheckManager.checkLinks(refDBInst, new ArrayList<>(LinksToCheckCache.removeRefDBFromCache(refDBInst)), this.proportionToLinkCheck, this.maxNumberLinksToCheck);
+		List<GKInstance> instancesToCheck = new ArrayList<>(LinksToCheckCache.removeRefDBFromCache(refDBInst));
```

Comment:
Just curious about what this line in particular its doing -- its both removing refDBs from cache and generating a list of instances to check?

Reply:
It calls the `remove` method on the underlying map. The `remove` method is like `get`, but the data is removed from the map.